stdlib/test/builtin/test_simd.mojo

```diff
@@ -42,18 +39,11 @@ def test_cast():
     )
 
 
-# CHECK-LABEL: test_simd_variadic
-fn test_simd_variadic():
-    print("== test_simd_variadic")
-
-    # CHECK: [52, 12, 43, 5]
-    print(SIMD[DType.index, 4](52, 12, 43, 5))
+def test_simd_variadic():
+    assert_equal(str(SIMD[DType.index, 4](52, 12, 43, 5)), "[52, 12, 43, 5]")
```

Comment:
**Suggestion** How about we actually use `SIMD.eq` rather than `str()` on both sides essentially?  `str` was only there for `FileCheck` since it's just a fancy regex matcher in short.  We can actually call real methods and not just do string matching shenanigans now that we're using the `testing` module.

Reply:
For this case in particular, I don't think switching would make sense. We would end up with 
```
assert_equal(SIMD[DType.index, 4](52, 12, 43, 5), SIMD[DType.index, 4](52, 12, 43, 5))
```
which seems kind of pointless as a test. To verify that the numbers are correctly stored in the SIMD, I think converting to string is a valid thing to do

Reply:
Oh, I see now - thanks for pointing that out.  The whole test seems pretty low value, honestly, but we can keep it for now; not a biggie.  We're in the process of syncing repos right now to push out a new `nightly/mojo`, so I'm holding off from merging this, but I'll merge it later today.

Reply:
Sure thing!